sphinx/source/docs/user_guide/bokehjs.rst

```diff
@@ -337,3 +337,63 @@ function, with the plot it generates shown below:
 
 .. image:: /_images/bokehjs_bar_charts.png
     :width: 100%
+
+
+``Minimal Complete Example``
+''''''''''''''''''''''''''''
+
+A minimal example follows, demonstrating a proper import of the libraries,
+and dynamic creation and modification of plots.
+
+.. code-block:: html
+
+    <!doctype html>
+    <html lang="en">
+    <head>
+    <meta charset="utf-8">
+    <title>Complete Example</title>
+    <link rel="stylesheet" href="http://cdn.pydata.org/bokeh/release/bokeh-0.12.5.min.css" type="text/css" />
+    <script type="text/javascript" src="http://cdn.pydata.org/bokeh/release/bokeh-0.12.5.min.js"></script>
+    <script type="text/javascript" src="http://cdn.pydata.org/bokeh/release/bokeh-api-0.12.5.min.js"></script>
```

Comment:
CDN links should be `https` (this will work but it's always better to demonstrate with `https`)